hazelcast/src/main/resources/icmp_helper.h

```diff
@@ -2,8 +2,8 @@
 #include <jni.h>
 /* Header for class com_hazelcast_util_ICMPHelper */
```

Comment:
Let's remove this comment or change the package on this line too.

Reply:
Changed package name.